crates/storage-api/src/invocation_status_table/mod.rs

```diff
@@ -242,6 +242,7 @@ impl InboxedInvocation {
 
     pub fn append_response_sink(&mut self, new_sink: ServiceInvocationResponseSink) {
         self.response_sinks.insert(new_sink);
+        self.timestamps.update();
     }
```

Comment:
Is this method used somewhere? Couldn't find any usages. Maybe better to delete if this is the case.

Reply:
Removed